featurebyte/schema/use_case.py

```diff
@@ -17,7 +17,8 @@ class UseCaseCreate(FeatureByteBaseModel):
 
     id: Optional[PydanticObjectId] = Field(default_factory=PydanticObjectId, alias="_id")
     name: StrictStr
-    target_id: PydanticObjectId
+    target_id: Optional[PydanticObjectId]
+    target_namespace_id: Optional[PydanticObjectId]
```

Comment:
same here - can namespace id be non optional?

Reply:
Here as an input, we need to make both optional. But i will move the validator here that either of them must be set